Makefile

```diff
@@ -61,6 +61,9 @@ build-ts: ## build TypeScript application
 	@$(YARN) build
 .PHONY: build-ts
 
+compiledmessages: ## compile the gettext files
```

Comment:
Why not calling this rule `compilemessages` to stick with Django management commands?

Reply:
True this one. @mbenadda I let you change so we don't step on each other's commits?

Reply:
Yeah I'll do it, I'm updating the PR progressively through several rounds of `rebase -i` so it's definitely better if we don't do it concomitantly